### DISCUSSION TITLE
How to disable friendly-snippets?

### DISCUSSION THREAD

**Discussion by Syakhisk** [LunarVim/LunarVim#2672] [Q&A] [answered]
I'm planning to write all my snippets, how to disable the default friendly-snippets?

**Comment by boydaihungst** [comment, marked as answer]
https://github.com/LunarVim/LunarVim/pull/2660
Put this in your `config.lua` file

```
lvim.builtin.friendly_snippets.active = false
```

**Comment by anshiq** [comment]
i want to friendly_snippets but want to  add more snippets but in my .config/lvim directory how can i link them in my lvim configuration
